### ISSUE TITLE
only the first POD is queried for clusters during addHost dialog

### ISSUE THREAD

**Issue by DaanHoogland** [apache/cloudstack#5864]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
4.16/snapshot
~~~

##### STEPS TO REPRODUCE

~~~
In an environment with two PODs and each a cluster.
add Host and select the second POD
~~~

##### EXPECTED RESULTS

~~~
clusters of the second pod are shown in the cluster dropdown
~~~

##### ACTUAL RESULTS

~~~
clusters of the first POD are shown in the cluster dropdown
~~~

**Comment by sureshanaparti** [comment]
Fixed in #5869

**Comment by sureshanaparti** [comment]
thanks @utchoang for fixing this issue.
